src/cmds/scripts/pbs_init.d.in

```diff
@@ -145,53 +144,53 @@ check_started() {
 : check_prog
 check_prog() {
 
-  case $1 in
-    mom)
-        daemon_name="pbs_mom"
-        pid=${pbs_mom_pid} ;;
-    server)
-        daemon_name="pbs_server.bin"
-        if [ ${is_secondary} -eq 0 ] ; then
-            pid=${pbs_server_pid}
-        else
-            pid=${pbs_secondary_server_pid}
-        fi ;;
-    oldserver)
-        daemon_name="pbs_server"
-        if [ ${is_secondary} -eq 0 ] ; then
-            pid=${pbs_server_pid}
-        else
-            pid=${pbs_secondary_server_pid}
-        fi ;;
-    sched)
-        daemon_name="pbs_sched"
-        if [ ${is_secondary} -eq 0 ] ; then
-            pid=${pbs_sched_pid} ;
-        else
-            pid=${pbs_secondary_sched_pid} ;
-        fi ;;
-    pbs_comm)
-        daemon_name="pbs_comm"
-        if [ ${is_secondary} -eq 0 ] ; then
-            pid=${pbs_comm_pid} ;
-        else
-            pid=${pbs_secondary_comm_pid} ;
-        fi ;;
-
-    *)   echo Invalid PBS daemon name: $1 >&2;
-	 return 1;;
-  esac
-
-  if [ -n "${pid}" ]; then
-    if [ "${pid}" -ne -1 ] ; then
-      if check_started "${pid}" "$daemon_name" ; then
-        return 0
-      fi
-    fi
-  fi
+	case $1 in
+		mom)
+			daemon_name="pbs_mom"
+			pid=${pbs_mom_pid} ;;
+		server)
+			daemon_name="pbs_server.bin"
+			if [ ${is_secondary} -eq 0 ] ; then
+				pid=${pbs_server_pid}
+			else
+				pid=${pbs_secondary_server_pid}
+			fi ;;
+		oldserver)
+			daemon_name="pbs_server"
+			if [ ${is_secondary} -eq 0 ] ; then
+				pid=${pbs_server_pid}
+			else
+				pid=${pbs_secondary_server_pid}
+			fi ;;
+		sched)
+			daemon_name="pbs_sched"
+			if [ ${is_secondary} -eq 0 ] ; then
+				pid=${pbs_sched_pid} ;
+			else
+				pid=${pbs_secondary_sched_pid} ;
+			fi ;;
+		pbs_comm)
+			daemon_name="pbs_comm"
+			if [ ${is_secondary} -eq 0 ] ; then
+				pid=${pbs_comm_pid} ;
+			else
+				pid=${pbs_secondary_comm_pid} ;
+			fi ;;
+
+		*)	echo Invalid PBS daemon name: $1 >&2;
+			return 1;;
+	esac
+
+	if [ -n "${pid}" ]; then
+		if [ "${pid}" -ne -1 ] ; then
+			if check_started "${pid}" "$daemon_name" ; then
```

Comment:
Also can you please change this kinda syntax to `if []; then`? or others to this one. I mean we should use one syntax.